django-stubs/views/static.pyi

```diff
@@ -3,7 +3,7 @@ from typing import Any, Optional
 from django.http.request import HttpRequest
 from django.http.response import FileResponse
 
-def serve(request: HttpRequest, path: str, document_root: str = ..., show_indexes: bool = ...) -> FileResponse: ...
+def serve(request: HttpRequest, path: str, document_root: Optional[str] = ..., show_indexes: Optional[bool] = ...) -> FileResponse: ...
```

Comment:
Thanks! But, looks like `show_indexes` is `bool`: https://github.com/django/django/blob/master/django/views/static.py#L19

Reply:
Sorry, you're right. I'm fairly new to type hints.